library/src/main/java/com/premnirmal/Magnet/WindowManagerHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.premnirmal.Magnet;
+
+import android.os.Build;
+import android.view.WindowManager;
+
+/**
+ * Created by nishant.patel on 18/12/17.
+ * Class with some helper methods to assist with WindowManager
+ */
+
+public class WindowManagerHelper {
```

Comment:
prefer not to have a separate class for this. can you move this to the `addToWindow()` methods